.github/workflows/terraspace.yaml

```diff
@@ -11,6 +11,7 @@ on:
     paths:
       - terraspace/**.tf
       - terraspace/**.tfvars
+      - .github/**
```

Comment:
Dunno if we want to trigger a TF plan each time _any_ github workflow has been modified.

Reply:
Changed for triggering only when changes happened in this workflow